_security-plugin/multi-tenancy/mt-agg-view.md

```diff
@@ -0,0 +1,74 @@
+
+### Known limitations
+
+In this first experimental phase of development, there are some limitations that should be observed before enabling the feature and using it in a test environment.
+
+* The feature can only be used in a new cluster. At this time, the feature is not suported by clusters already in use.
+* Also, the feature should be used only in a test environment, not in production. 
+* Finally, once the feature has been enabled and used in a test cluster, the feature cannot be disabled for the cluster. Disabling the feature once it has been used to work with tenants and saved objects can result in the loss of saved objects and have an impact on tenant-to-tenant functionality. 
+
+These limitations will be addressed in upcoming releases.
+
+## Enabling aggregate view for saved objects
+
+By default, the aggregate view in the Saved Objects table is disabled. To enable the feature, add the `opensearch_security.multitenancy.enable_aggregation_view` flag to the `opensearch_dashboards.yml` and set it to `true`.
+
+`opensearch_security.multitenancy.enable_aggregation_view: true`
+
+After enabling the feature you can start the new cluster and then launch Dashboards. 
+
+## Working in aggregate view
+
+Select the drop-down arrow on the Tenants column to display the list of tenants available to the user. You can select multiple tenants while the menu is open.
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/tenant_column.png" alt="Dashboards Saved Objects view with emphasis on Tenants column" width="700">
+
+Select anywhere outside the menu to collapse it. The Title column displays the saved objects corresponding to the selected tenants. The Tenants column displays the tenants associated with the saved objects. Also, the number of tenants selected for filtering is expressed in a red box on the drop-down.
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/ten-filter-results.png" alt="Dashboards Saved Objects tenant filtering" width="700">
+
+Use the Type column to filter saved objects by type. The behavior for the drop-down is the same as the Tenants column drop-down.
+
+### Selecting and working with a saved object
+
+1. After identifying a saved object that you would like to work with, note the tenant associated with the object in the Tenants column. 
+1. In the top right corner of the window, open the user menu and select **Switch tenants**.
```

Comment:
Use "upper right" and "lower right" instead of "top" and "bottom", and hyphenate as an adjective.